lighthouse/blueprints/cherrypicked_plates.py

```diff
@@ -28,14 +28,18 @@
 CORS(bp)
 
 
-@bp.route("/cherrypicked-plates/create", methods=["POST"])
+@bp.route("/cherrypicked-plates/create", methods=["GET"])
 def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
+
+    invalid_url = {"errors": ["GET request needs 'barcode' in url"]}, HTTPStatus.BAD_REQUEST
```

Comment:
Maybe we can move this to a helper function inside this module? Instead of keeping it stored in a variable, giving it back as the response of a function; the reason is, we don't want to keep it in memory in the cases when it is not going to be in use